Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -742,6 +744,82 @@ public function getPrevMeaningfulToken($index)
     }
 
     /**
+     * Find a sequence of meaningful tokens and returns the array of their locations.
+     *
+     * @param array      $sequence      an array of tokens (same format used by getNextTokenOfKind)
+     * @param int        $start         start index, defaulting to the start of the file.
+     * @param int        $end           end index, defaulting to the end of the file.
+     * @param bool|array $caseSensitive global case sensitiveness or an array of booleans (for each token)
+     *
+     * @return array|null an array containing the indexes for the sequence elements
```

Comment:
the Tokens interfce for such cases is to return assoc array that mapping `index => token`